lint

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-export PYTHONPATH='sigopt'
```

Comment:
Tbh I did not know if there was some convention around this that I was supposed to follow. I'll undo this change!

Sent from my iPhone

> On Jul 11, 2016, at 6:50 PM, Patrick Hayes <notifications@github.com> wrote:
> 
> In lint:
> 
> > @@ -1,7 +1,5 @@
> >  #!/usr/bin/env bash
> >  
> > -export PYTHONPATH='sigopt'
> why not leave this in here so the script works on its own?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>